recipes/default.rb

```diff
@@ -41,6 +41,23 @@
   action :nothing
 end.run_action(:run)
 
+# Prepare the environment variables
+$gecos_environ = ENV.inject({}){|memo,(k,v)| memo[k.to_sym] = v; memo}
+
+# Get workstation name from /etc/pclabel
```

Comment:
Hi Alfonso, I performed some tests and you can't trust OHAI data when this recipe is launched.
It seems that the information of the node starts empty and then is filled with OHAI data when the gecos-workstation-ohai-cookbook recipes are run. But you don't know the order in witch both cookbooks will be executed, also you can't share information between both cookbooks.

I suppose that pclabel could be read from OHAI data if a dependency between both cookbooks is specified in the "metadata.rb" file. But this is just a guess.
  https://docs.chef.io/config_rb_metadata.html (see "depends" in Settings section)

Best regards!

Reply:
AFAIK gecos-workstation-ohai-cookbook only put ohai plugins in place.

Then, standard chef run executes ohai plugins before launching recipes.

I think your tests show an abnormal behaviour that needs further investigation.

Sorry for been a bit 'picky' but I just try to avoid dirty/unexplained code in the project.